roles/kubernetes/secrets/tasks/gen_certs_vault.yml

```diff
@@ -54,6 +54,10 @@
         {%- endfor -%}
         "127.0.0.1","::1","{{ kube_apiserver_ip }}"
         ]
+        {%- if supplementary_addresses_in_ssl_keys is defined -%}
```

Comment:
According to my understanding the concern is that outcome of snippet will be lists of lists instead of just list. However this will work more or less like Python's extend. You can check it by running example  `ansible-playbook demo-lists.yml` from: https://github.com/towasz/ansible-examples